src/shellingham/posix/__init__.py

```diff
@@ -4,34 +4,26 @@
 from .._core import SHELL_NAMES, ShellDetectionFailure
 from . import proc, ps
 
+QEMU_BIN_REGEX = re.compile(
+    r"qemu-(alpha|armeb|arm|m68k|cris|i386|x86_64|microblaze|mips|mipsel|mips64|mips64el|mipsn32|mipsn32el|nios2|ppc64|ppc|sh4eb|sh4|sparc|sparc32plus|sparc64)"
+)
```

Comment:
Let’s split this string into multiple lines with `re.VERBOSE` for readability. Would be a good idea to add a comment linking to QEMU docs for future reference as well.